projects/rocprofiler-sdk/source/include/rocprofiler-sdk/experimental/thread-trace/core.h

```diff
@@ -53,8 +66,9 @@ typedef enum rocprofiler_thread_trace_parameter_type_t
     ROCPROFILER_THREAD_TRACE_PARAMETER_PERFCOUNTER_EXCLUDE_MASK,  ///< Bitmask of which compute
                                                                   ///< units to exclude from
                                                                   ///< perfcounters. gfx9 only
-    ROCPROFILER_THREAD_TRACE_PARAMETER_NO_DETAIL,  ///< Dont collect instruction timing,
-                                                   ///< only shader-wide information
+    ROCPROFILER_THREAD_TRACE_PARAMETER_NO_DETAIL,       ///< Dont collect instruction timing,
+                                                        ///< only shader-wide information
+    ROCPROFILER_THREAD_TRACE_PARAMETER_BUFFERING_MODE,  ///< Enables GPU Triple buffering.
```

Comment:
I though double buffering and triple buffering are specified with the number of buffers, so maybe this comment is more appropriate.

```suggestion
    ROCPROFILER_THREAD_TRACE_PARAMETER_BUFFERING_MODE,  ///< Enables GPU double/triple buffering.
```

Reply:
Today dont have double buffering support in the GPU, and and I want to add triple in the CPU as well.